.github/workflows/ci.yml

```diff
@@ -31,6 +31,7 @@ jobs:
           - ubuntu-20.04
           - macos-10.15
```

Comment:
Testing two macos builds is unnecessary. This just adds more pressure on GitHub's CI servers.

```suggestion
```